README.md

```diff
@@ -1 +1,31 @@
-# AwesomeBooks
\ No newline at end of file
+# AwesomeBooks
+ 
+A simple JavaScript Project that display bok Title and Author. When a user click on the add button, you have the value imputed added at the top of the page and when user clicks on Remove button it automatically remove the value input. Modified the old code since it already match the project requirement.
+
+Added Modules and made use of ES6
+
+
+# Build with 
+ - HTML 
+ - CSS
+ - JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://sirriryisa.github.io/Capstone-proj/)
+
+
+👤 **Author**
+- GitHub: [@githubhandle](https://github.com/SirriRyisa)
+- Twitter: [@twitterhandle](https://twitter.com/N_Ryisa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ryisa-sirri-ngwa-a30013202)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
```

Comment:
- [ ] Kindly add `issues page link` and `license`. Because these will make it possible to get feedback from others.